### DISCUSSION TITLE
Manual plugins

### DISCUSSION THREAD

**Discussion by alfredbuehler** [aljawaid/PluginManager#37] [General]
https://github.com/kanboard/website/pull/428

@aljawaid Did you start adding other peoples plugins to the repo? I strongly hope you ask them before you do.

BTW, something is strange with the forum at now, therefore I'm asking here.

**Comment by aljawaid** [comment]
Hello I only added tagus plugins with his permission a few weeks ago

Then I added Frederics budget plugin which is archived.

Everything else in manual plugins that is listed was already present

**Comment by alfredbuehler** [comment]
Thank you, good to know!
